os_dep/linux/ioctl_cfg80211.c

```diff
@@ -1080,7 +1080,13 @@ void rtw_cfg80211_indicate_connect(_adapter *padapter)
 		#endif
 
 		#if LINUX_VERSION_CODE >= KERNEL_VERSION(4, 12, 0)
+		#if LINUX_VERSION_CODE < KERNEL_VERSION(6, 0, 0)
+		roam_info.channel = notify_channel;
```

Comment:
This line is changing functionality for older kernels, are you sure this is intended?

Reply:
It is based on https://github.com/lwfinger/rtl8188eu/commit/0a5e81555c5e9dc16030598d4e11f8f1dad541be#diff-d1818f3f2b9ced9fbfa5877b46653beda3257d3f6528036bdd63e302a4420e4cR709
I take your point if the existing code is working without why add it?